ansible/.env.j2

```diff
@@ -0,0 +1,16 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Required to use any Trello API
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_API_TOKEN={{trello_api_token}}
+BOARD_ID=65cf51297e3196bc2180b589
+TODO_LIST_ID=65cf512945f27eeecfb5e6b8
+DONE_LIST_ID=65cf512a5cb54429e2fd70cc
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)